src/components/widgets/sparkline/sparkline_dataHelpers.js

```diff
@@ -0,0 +1,88 @@
+
+// import Highcharts from 'highcharts';
+import last from 'lodash/last';
+
+
+export const makeChartOptions = ({
+  emitSetState = () => {},
+  widget,
+}) => {
+  return {
+    // default options
+    chart: {
+      type: 'spline',
+      margin: [150, 0, 0, 0],
+      events: {
+        load: function() {
+          var latestValue = last(this.series[0].data).y
+          this.renderer.text(latestValue)
```

Comment:
do you need to create from constructor ie `this.renderer = new Renderer()` or `this.renderer` implies that under the hood? Does a chart always have one renderer available?